.github/workflows/pr.yml

```diff
@@ -13,9 +13,9 @@ jobs:
       OPERATOR_SDK_VERSION: v1.22.0
     steps:
       - name: Set up Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v4
         with:
-          go-version-file: go.mod
+          go-version: 1.22
```

Comment:
isn't it better to read the version from the go.mod file instead of being pinned and having to update multiple places when we switch go versions?

Reply:
I agree, but it reported the go.mod file couldn't be found with actions/setup-go@v4, even it's placed to other directory.  And it failed with  actions/setup-go@v2 since the golang version is low.

Reply:
It reported go.mod file can't be found either with v5.

Reply:
Can you change the PR to use the file with v5 and leave it so I can look at the logs? It is already back to go 1.22 hardcoded. The release-service uses this same action with v5 and a go.mod file, so idk how that could be

Reply:
If you want to debug it, can you pls. create a PR by yourself? Because you may want do some change several times. It will be slow to do things like that.

Reply:
I mean I guess but I am not approving this PR with the go version tied to 1.22

Reply:
Filed a ticket https://issues.redhat.com/browse/RELEASE-1756 for it

Reply:
Updated my PR.